circus-train-core/src/main/java/com/hotels/bdp/circustrain/core/replica/ReplicaTableFactory.java

```diff
@@ -46,19 +47,20 @@
 public class ReplicaTableFactory {
 
   /**
-   * Parameter to indicate we wish to retain table statistics >= Hive 2.0.0. See <a href=
-   * "https://github.com/apache/hive/blob/release-2.0.0/common/src/java/org/apache/hadoop/hive/common/StatsSetupConst.java#L120">
-   * hive-exec:2.0.0#StatsSetupConst:120</a>.
+   * Parameter to indicate we wish to retain table statistics >= Hive 2.0.0.
    */
-  private static final String DO_NOT_UPDATE_STATS = "DO_NOT_UPDATE_STATS";
+  private static final String DO_NOT_UPDATE_STATS = StatsSetupConst.DO_NOT_UPDATE_STATS;
```

Comment:
I think this is going to fail is the libraries use an old hive version

Reply:
I've tested this using an older version of Hive (1.2.1) and it's fine. This is because the strings are static final and will be replaced at *compile* time not run time (see https://stackoverflow.com/questions/5173372/java-static-final-values-replaced-in-code-when-compiling)

Reply:
truth :)